packages/sdk/package.json

```diff
@@ -4,52 +4,33 @@
   "version": "65.2.0",
   "author": "Linear Orbit, Inc",
   "license": "MIT",
-  "main": "dist/index-cjs.min.js",
-  "module": "dist/index-es.min.js",
-  "unpkg": "dist/index-umd.min.js",
-  "types": "dist/index.d.ts",
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "default": "./dist/index-cjs.min.js"
-    },
-    "./webhooks": {
-      "types": "./dist/webhooks/index.d.ts",
-      "default": "./dist/webhooks/index-cjs.min.js"
-    }
-  },
   "repository": "https://github.com/linear/linear",
```

Comment:
Syntax here seems different now [https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository).

edit: I see this was the existing value, so maybe can do in a followup PR just to minimize number of things changing here haha